### PR TITLE
Updated german strings

### PR DESCRIPTION
**Pull request by failex234**

### CODE REVIEW

File: app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -338,6 +338,15 @@
     <string name="successfully_invited">Erfolgreich eingeladen!</string>
     <string name="reply">Antworten</string>
     <string name="failed_selecting_image">Bild konnte nicht geladen werden.</string>
+	<string name="want_to_merge">möchte mergen</string>
+    <string name="watchers">Betrachter</string>
+    <string name="reviewers">Reviewer</string>
+    <string name="no_reviewers">Keine Reviewer</string>
+    <string name="sent_from_fasthub">Gesendet von meinem %1$s mit %2$s%3$s</string>
+    <string name="enable_signature">Signatur aktivieren</string>
+    <string name="enable_signature_summary">&quot;Gesendet via&quot;-Signatur aktivieren</string>
```
Comment (FSMaxB) [inline app/src/main/res/values-de/strings.xml]: This should use `Gesendet von` because the actual signature is `Gesendet von meinem ...`

Comment (failex234) [inline app/src/main/res/values-de/strings.xml]: Yeah, you're right, thank you!

File: app/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -338,6 +338,15 @@
     <string name="successfully_invited">Erfolgreich eingeladen!</string>
     <string name="reply">Antworten</string>
     <string name="failed_selecting_image">Bild konnte nicht geladen werden.</string>
+	<string name="want_to_merge">möchte mergen</string>
```
Comment (sschuberth) [inline app/src/main/res/values-de/strings.xml]: Indentation looks off here. (tab vs spaces?)

**Comment by k0shk0sh** [comment]
thanks @failex234

**Pull request merged**